lib/Parser/Scan.h

```diff
@@ -667,7 +667,7 @@ class Scanner : public IScanner, public EncodingPolicy
     BOOL m_doubleQuoteOnLastTkStrCon :1;
     bool m_OctOrLeadingZeroOnLastTKNumber :1;
     BOOL m_fSyntaxColor : 1;            // whether we're just syntax coloring
```

Comment:
This pattern of mixing types on bit fields might be bloating our struct sizes: http://stackoverflow.com/questions/308364/c-bitfield-packing-with-bools